greengrass_ipc/source/GreengrassCoreIpcModel.cpp

```diff
@@ -5933,6 +7016,253 @@ namespace Aws
+            noexcept
+        {
+            (void)stringView;
+            (void)allocator;
+            return nullptr;
+        }
+
+        Aws::Crt::String VerifyClientDeviceIdentityOperationContext::GetRequestModelName() const noexcept
+        {
+            return Aws::Crt::String("aws.greengrass#VerifyClientDeviceIdentityRequest");
+        }
+
+        Aws::Crt::String VerifyClientDeviceIdentityOperationContext::GetInitialResponseModelName() const noexcept
+        {
+            return Aws::Crt::String("aws.greengrass#VerifyClientDeviceIdentityResponse");
+        }
+
+        Aws::Crt::Optional<Aws::Crt::String> VerifyClientDeviceIdentityOperationContext::GetStreamingResponseModelName()
+            const noexcept
+        {
+            return Aws::Crt::Optional<Aws::Crt::String>();
+        }
+
+        Aws::Crt::String VerifyClientDeviceIdentityOperationContext::GetOperationName() const noexcept
+        {
+            return Aws::Crt::String("aws.greengrass#VerifyClientDeviceIdentity");
+        }
+
+        std::future<VerifyClientDeviceIdentityResult> VerifyClientDeviceIdentityOperation::GetResult() noexcept
+        {
+            return std::async(std::launch::deferred, [this]() {
+                return VerifyClientDeviceIdentityResult(GetOperationResult().get());
+            });
+        }
+
+        VerifyClientDeviceIdentityOperation::VerifyClientDeviceIdentityOperation(
+            ClientConnection &connection,
+            const VerifyClientDeviceIdentityOperationContext &operationContext,
+            Aws::Crt::Allocator *allocator) noexcept
+            : ClientOperation(connection, nullptr, operationContext, allocator)
+        {
+        }
+
+        std::future<RpcError> VerifyClientDeviceIdentityOperation::Activate(
+            const VerifyClientDeviceIdentityRequest &request,
+            OnMessageFlushCallback onMessageFlushCallback) noexcept
+        {
+            return ClientOperation::Activate(static_cast<const AbstractShapeBase *>(&request), onMessageFlushCallback);
+        }
+
+        Aws::Crt::String VerifyClientDeviceIdentityOperation::GetModelName() const noexcept
+        {
+            return m_operationModelContext.GetOperationName();
+        }
+
+        AuthorizeClientDeviceActionOperationContext::AuthorizeClientDeviceActionOperationContext(
+            const GreengrassCoreIpcServiceModel &serviceModel) noexcept
+            : OperationModelContext(serviceModel)
+        {
+        }
+
+        Aws::Crt::ScopedResource<AbstractShapeBase> AuthorizeClientDeviceActionOperationContext::
+            AllocateInitialResponseFromPayload(Aws::Crt::StringView stringView, Aws::Crt::Allocator *allocator) const
+            noexcept
+        {
+            return AuthorizeClientDeviceActionResponse::s_allocateFromPayload(stringView, allocator);
+        }
+
+        Aws::Crt::ScopedResource<AbstractShapeBase> AuthorizeClientDeviceActionOperationContext::
```

Comment:
Is this supposed to return a nullptr? This seems like potentially missing functionality...